src/components/tx/TxData.js

```diff
@@ -279,6 +279,19 @@ class TxData extends React.Component {
           return renderNanoContractMatchValues(output.decoded);
         default:
           let script = output.script;
+          // Try to parse as script data
+          try {
+            // The output script is decoded to base64 in the full node
+            // before returning as response to the explorer in the API
+            // and the lib expects a buffer (bytes)
+            // In the future we must receive from the full node
+            // the decoded.type as script data but this still needs
+            // some refactor there that won't happen soon
+            const buff = new Buffer.from(script, 'base64');
+            const parsedData = hathorLib.scriptsUtils.parseScriptData(buff);
+            return renderDataScript(parsedData.data);
+          } catch {}
```

Comment:
This exception handler might mask bugs in the parsing script. Shouldn't we capture a specific type of error here?

Reply:
We receive the output script from the full node in base64.

The old behaviour of this parse method was to try to decode the base64 and show 'Unable to decode: X' and it would show X as the decoded base64 or whatever the full node has sent to us as output script. I tried to keep the same idea, if we can't parse the script we will show 'Unable to decode: X' where X is what we received from the full node.

I understand that the best approach is usually to capture the specific error but I guess the idea here is to prevent any exception in the explorer and show as 'Unable to decode', as the previous code used to do.

Reply:
I understand. But a bug is different from an unknown script. We should at least log the error in case of a bug.

Reply:
Makes sense. Added the handling and log here https://github.com/HathorNetwork/hathor-explorer/pull/141/commits/66d891268e9f16682e776c0e7b8e116ff90be3db